txt2tags.py

```diff
@@ -1189,6 +1195,60 @@ def getTags(config):
             # TODO: placeholder (mark for unknown syntax)
             # if possible: http://www.wikicreole.org/wiki/Placeholder
         },
+            # regular markdown: http://daringfireball.net/projects/markdown/syntax
+            # markdown extra:   http://michelf.com/projects/php-markdown/extra/
+        "md": {
+            "title1"               : "# \a "         ,
+            "title2"               : "## \a "        ,
+            "title3"               : "### \a "       ,
+            "title4"               : "#### \a "      ,
+            "title5"               : "##### \a "     ,
+            "blockVerbLine"        : "    "          ,
+            "blockQuoteLine"       : "> "            ,
+            "fontMonoOpen"         : "`"             ,
+            "fontMonoClose"        : "`"             ,
+            "fontBoldOpen"         : "**"            ,
+            "fontBoldClose"        : "**"            ,
+            "fontItalicOpen"       : "*"             ,
+            "fontItalicClose"      : "*"             ,
+            "fontUnderlineOpen"    : ""              ,
+            "fontUnderlineClose"   : ""              ,
+            "fontStrikeOpen"       : "~~"            ,
+            "fontStrikeClose"      : "~~"            ,
+            # Lists
+            #"listOpenCompact"     : "*"             ,
+            "listItemLine"         : " "             ,
+            "listItemOpen"         : "*"             ,
+            #"numlistItemLine"     : "1."            ,
+            "numlistItemOpen"      : "1."            ,
+            "deflistItem1Open"     : ": "            ,
+            #"deflistItem1Close"     : ":"           ,
+            #"deflistItem2LineOpen"  : "::"          ,
+            #"deflistItem2LineClose" : ":"           ,
+            # Verbatim block
+            #"blockVerbOpen"        : ""             ,
+            #"blockVerbClose"       : ""             ,
+            "bar1"                 : "---"           ,
+            "bar2"                 : "---"           ,
+            # URL, email and anchor
+            "url"                   : "\a"           ,
+            "urlMark"               : "[\a](\a)"     ,
+            "email"                 : "\a"           ,
+            #"emailMark"             : "[[\a -> mailto:\a]]",
+            #"anchor"                : "[[#\a]]\n"   ,
+            # Image markup
+            "img"                   : "![](\a)"      ,
+            #"imgAlignLeft"         : "{{\a }}"      ,
+            #"imgAlignRight"        : "{{ \a}}"      ,
+            #"imgAlignCenter"       : "{{ \a }}"     ,
```

Comment:
I don't think we should add commented-out code (here and above). It is not clear why the code is commented out. I believe it serves as a placeholder for tags that might be supported later, but then I'd prefer to keep only the left parts (e.g., imgAlignLeft) and not the right parts (e.g., "{{\a }}") since the latter apparently don't "work" in Markdown.

For the long term, I think we should us `None` for unsupported tags to avoid the commented-out code, but this should be a separate PR, of course.

Reply:
I don't know either. I can remove it right now, and it will forgotten for ever, or we could modify this later (it will be easier this way I suppose)